src/authInfo.ts

```diff
@@ -758,8 +759,7 @@ export class AuthInfo extends AsyncCreatable<AuthInfo.Options> {
     this.fields.username = this.options.username || getString(options, 'username') || undefined;
```

Comment:
Is last `|| undefined` necessary, given getString either returns the property or undefined?

Reply:
I agree that the `|| undefined` suffix looks unnecessary based on how `getString` works

Reply:
TS complains:
_Type 'Optional<string | null>' is not assignable to type 'string | undefined'.
  Type 'null' is not assignable to type 'string | undefined'._

going to leave it for now

Reply:
Ahh right.  `getString` returns `string | null` not `string | undefined`.  null !== undefined